src/hooks/useThrowError.tsx

```diff
@@ -1,21 +1,25 @@
 import * as React from 'react';
+import { useSelector } from 'react-redux';
 
-type Props = {
-    inputValue: string
+// Interface
+import { RootState } from '../types/redux.interface';
+
+type Error = {
+    error: Error | any
```

Comment:
This type doesn’t have any sense. So it can be anything. Cannot it be ? 
```suggestion
    error: Error
```

Reply:
resolved